src/plugins/intel_cpu/src/nodes/gather.cpp

```diff
@@ -585,6 +609,46 @@ void Gather::exec1DCase() {
     }
 }
 
+void Gather::execCompressedEmbedding() {
```

Comment:
minor: the domain perspective we can compress not only embeddings, so better to use more abstract name (like execCompressedCase);

Reply:
Done